Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -1443,7 +1443,7 @@ static void normalize_script_extensions(PropList& script_extensions, PropList co
     // abbreviation. Further, the extensions may or may not include the base Script list. Normalize
     // the extensions here to be keyed by the full Script name and always include the base list.
     auto extensions = move(script_extensions);
-    script_extensions = script_list;
+    script_extensions = MUST(script_list.clone());
```

Comment:
`normalize_script_extensions` is only invoked in one function that returns an `ErrorOr`; maybe you can make this a `TRY` instead and return `ErrorOr<void>`?

Reply:
Absolutely! Weird, I could have sworn I checked that before writing "MUST".